scripts/on-prem-destroy.sh

```diff
@@ -52,8 +52,18 @@ sleep "$(( GRACE + PADDING ))"
 # This is done to ensure data safety, which is generally more valuable
 # than an automatic purge of all related StatefulSet resources.
 kubectl --namespace default delete pvc -l component=elasticsearch,role=data || true
+kubectl --namespace default delete pvc -l component=elasticsearch,role=master || true
 
-echo "kubectl get pvc --all-namespaces"
-kubectl get pvc --all-namespaces
-echo "kubectl get pv --all-namespaces"
-kubectl get pv --all-namespaces
+timeout=100
+
+# Attempt to wait for the PV to be destroyed before proceeding. Most of the time the PV's are deleted immediately,
```

Comment:
PV's are not deleted immediately with stateful sets.  That is why we have to clean them up.  Please update the comments.

Reply:
comment updated